JULIERAJ/KIDS-FIRST-Back
# Fix: update method for kid profile

One Line Description
Resolve issues in the updateKid method and enhance functionality to handle edits to a kid's profile updates.

Requirements
.  Fixed req.body handling and enhanced updateKid logic to address missing or empty fields.
.  Add support for profile picture uploads using Cloudinary.
.  Updated router.put to include middleware for file uploads and Cloudinary configuration.

Notes
.  Updates follow the logic from the createKid method for consistency.
.  Improvements include recalculating age, initializing arrays, and saving the profile picture URL.

Testing Instructions
Update Data:
      .  Test updating a kid’s profile with valid data (e.g., name, dateOfBirthday, allergies, etc.).
      .  Verify that the updated information is correctly saved and returned in the response.

API Changes 
-  PUT /:id: Updated to handle image uploads and dynamic field updates.

Checklist
